src/app/components/core/south/south.component.html

```diff
@@ -3,7 +3,7 @@
     <div id="scheduled-process" class="card-header-title">
       <div class="media-content">
         <p class="title is-5">
-          South Services &nbsp;
+          South Services
```

Comment:
Are we okay now without `nbsp;` ?

Reply:
yes, no issues. It was there to put gap between refresh button and title but now we don't have refresh button because of auto refresh functionality.